<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QDoubleSpinBox" name="number1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="number1Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your first number</string>
   </property>
  </widget>
  <widget class="QLabel" name="number2Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your second number</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="number2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="calculationType">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="resultLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="userResult">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Your result</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="result">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="tasksCounter">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tasksCounterLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of tasks solved</string>
   </property>
  </widget>
  <widget class="QLabel" name="tasksCorrectLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Solved correctly</string>
   </property>
  </widget>
  <widget class="QLabel" name="tasksIncorrectLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>solved incorrectly</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="statsLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="tasksCorrect">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="tasksIncorrect">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
